internal/restore/volumesnapshot_action.go

```diff
@@ -61,7 +61,7 @@ func resetVolumeSnapshotAnnotation(vs *snapshotv1api.VolumeSnapshot) {
 // Execute uses the data such as CSI driver name, storage snapshot handle, snapshot deletion secret (if any) from the annotations
 // to recreate a volumesnapshotcontent object and statically bind the Volumesnapshot object being restored.
 func (p *VolumeSnapshotRestoreItemAction) Execute(input *velero.RestoreItemActionExecuteInput) (*velero.RestoreItemActionExecuteOutput, error) {
-	p.Log.Info("Starting VolumeSnapshotRestoreItemAction")
+
```

Comment:
This log should not be removed, right ?

Reply:
Ok I see what you meant.. 
Misread https://github.com/openshift/velero-plugin-for-csi/pull/63#discussion_r1099317680